app/app.py

```diff
@@ -120,6 +143,29 @@ def api_contacts():
     return ("", 405)  # Method Not Allowed
 
 
+def get_event_info(person_id, date_time):
+    """
+    Helper method to get event from selected date
+    """
+    """
+    result = db.engine.execute(
+        "SELECT * FROM CONTACTS WHERE person_id = "
+        + person_id
+        + "AND date_time = "
+        + str(date_time)
+    )
+    """
+    result = db.engine.execute(
```

Comment:
Avoid using raw SQL

Reply:
Just removed the line
I haven't started on the filter clause to fetch event yet